### ISSUE TITLE
App crash with "internal server error" after NC update 29.0.4 --> 29.0.5

### ISSUE THREAD

**Issue by jacotec** [SergeyMosin/Appointments#538]
I've updated my Nextcloud to 29.0.5 today and unfortunately complaints started that no one can't book appointments anymore.

The form is shown correctly with the correct available slots, but when it's sent, the booking person only gets this:

![image](https://github.com/user-attachments/assets/5e99f668-42f5-4fb5-90d6-0654ef7a4d4f)

In the calendar, the requested slot is already allocated with the text "Available". No email is sent to the requesting person.

In the Nextcloud log, the only entry from Appointments is:

```
[appointments] Error: bad event data, uid: a77d7f0-f7d2eb-cb242c-9029ba-tm0753957
	from ? by -- at 21.08.2024, 16:05:11
```

**Comment by SergeyMosin** [comment]
Bug confirmed (NC v29.0.5 is effected). Fix will be available as soon as possible.
Thank you for the report.

**Comment by jacotec** [comment]
Thanks for the feedback @SergeyMosin

**Comment by SergeyMosin** [comment]
Fixed in v2.1.10

"Broken" appointments have no attendees and can be deleted.
![image](https://github.com/user-attachments/assets/877a94b1-b5d7-499d-87d0-cc865852d034)
